<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>info</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>637</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(59, 0, 89, 180);
color: rgb(255, 255, 255);
border-style: outset;
border-widht:2px;
border-radius:10px;
border-color:black;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
